torch/csrc/autograd/variable.h

```diff
@@ -22,7 +22,7 @@ struct Variable : std::enable_shared_from_this<Variable> {
     SavedVariable(const Variable& variable, Function* saved_for)
       : data(variable.data->clone_shallow())
       , grad_accumulator(variable.grad_accumulator)
-      , version(std::move(variable.version_counter->new_saved_ref()))
+      , version(variable.version_counter->new_saved_ref())
```

Comment:
Yes, but it's an rvalue and using `move` prevents copy elision (without it the compiler will just initialise this `version` in-place).